openpype/hosts/nuke/api/plugin.py

```diff
@@ -808,6 +812,14 @@ def __init__(self,
         self.log.info("File info was set...")
 
         self.file = self.fhead + self.name + ".{}".format(self.ext)
+        if ".{}".format(self.ext) not in VIDEO_EXTENSIONS:
+            filename = get_head_filename_without_hashes(
+                self.path_in, self.name)
+            self.file = filename
+            if ".{}".format(self.ext) not in self.file:
+                original_ext = filename.split(".")[-1]
```

Comment:
```suggestion
                original_ext = os.path.splitext(filename)[-1].strip(".")
```
why not to use os path function dedicated extension split?